src/docs/user_guide.md

```diff
@@ -4260,8 +4296,28 @@ encoder: parallel_cnn
 ... encoder parameters ...
 ```
 
-Distributed Training
-====================
+Distributed Execution Backends
+==============================
+
+The same Ludwig config / Python code that runs on your local machine can be executed remotely in a distributed manner with zero code changes. This distributed execution include preprocessing, training, and batch prediction. 
+
+In most cases, Ludwig will be able to automatically detect if you're running in an environment that supports distributed execution, but you can also make this explicit on the command line with the `--backend` arg or by providing a `backend` section to the Ludwig config YAML:
+
+```yaml
+backend:
+  type: local
+  data_format: parquet
```

Comment:
maybe we could call this `cache_format`?

Reply:
Updated. Will also change in Ludwig.